src/storage/build.ts

```diff
@@ -12,24 +12,36 @@ import type {
 export const isStorage = (obj: unknown): obj is AxiosStorage =>
   !!obj && !!(obj as Record<string, boolean>)['is-storage'];
 
-/** Returns true if this has sufficient properties to stale instead of expire. */
-export function canStale(value: CachedStorageValue): boolean {
+function hasUniqueIdentifierHeader(
+  value: CachedStorageValue | StaleStorageValue
+): boolean {
   const headers = value.data.headers;
 
   return (
     Header.ETag in headers ||
     Header.LastModified in headers ||
     Header.XAxiosCacheEtag in headers ||
-    Header.XAxiosCacheStaleIfError in headers ||
     Header.XAxiosCacheLastModified in headers
   );
 }
 
+/** Returns true if this has sufficient properties to stale instead of expire. */
+export function canStale(value: CachedStorageValue | StaleStorageValue): boolean {
```

Comment:
Should be:

```suggestion
export function canStale(value: CachedStorageValue): boolean {
```

Right?